platform/debian/do_debian_package.sh

```diff
@@ -38,20 +38,42 @@ if command_exists "${COMMAND}"; then
     {
         echo "Section: graphics"
         echo "Priority: optional"
-        echo "Depends: libsdl2-2.0-0"
+        echo "Depends: libsdl2-2.0-0, libc6 (>= 2.2.1)"
```

Comment:
Shouldn't that (probably) be 2.23? What's 2.2.1? :-)

Reply:
That's the first glibc 2 version with a stable API/ABI, IIRC ^^.

(Although, in practice, that's more like glibc 2.4).

Reply:
I had that lying around on some old debian source package from 2016 era  <\_/>

In theory source packages need to include `Depends: ${shlibs:Depends}` and a helper tool will replace them based on real requirements. Buth yeah 2.2.3 is what Xenial ships. I will update.